include/nrf_rpc/nrf_rpc_uart.h

```diff
@@ -59,17 +59,9 @@ typedef struct nrf_rpc_uart {
 	struct k_work_q rx_workq;
 } nrf_rpc_uart;
 
-/**
```

Comment:
These changes are not really needed as you still choose the UART to use in BLE and OT group definition. Exposing public `NRF_RPC_UART_TRANSPORT` doesn't make sense if you only declare the object for `DT_CHOSEN(zephyr_rpc_uart` i.e. enforce a single instance of the transport.